core/generated.go

```diff
@@ -153,6 +153,10 @@ func (m *generateCommon) shortName() string {
 	return m.Name()
 }
 
+func (m *generateCommon) outputName() string {
```

Comment:
I just realised that outputName on generateCommon is actually incorrect, as it shouldn't be supported by generateSource and transformSource (which have multiple outputs). It should really be on generateLibrary in gen_library.go.

Reply:
moved to gen_library.go